src/app/components/content-page/content-page.component.ts

```diff
@@ -1,30 +1,31 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
+import { faPlusSquare } from '@fortawesome/free-solid-svg-icons';
 import { faSearch } from '@fortawesome/free-solid-svg-icons';
 
-import { ActivatedRoute } from '@angular/router';
-
 import { User } from 'src/app/models/user';
```

Comment:
User model not being used. Put there for future use ?

Reply:
believe that was @DVTWerries 's code

Reply:
removed 👍